packages/react-component-library/src/components/SectionDivider/SectionDivider.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { StyledSectionDivider } from './partials/StyledSectionDivider'
+import { StyledTitle } from './partials/StyledTitle'
+import { StyledDescription } from './partials/StyledDescription'
+import { ComponentWithClass } from '../../common/ComponentWithClass'
+
+export interface SectionDividerProps extends ComponentWithClass {
+  title?: string
+  children?: string
+}
+
+export const SectionDivider: React.FC<SectionDividerProps> = ({
```

Comment:
I wonder if another name would be better if this can have a title and description. Maybe something like `SectionHeading`?